### PR TITLE
Aditya task develop UI

### PR DESCRIPTION
**Pull request by saurabh-ifmt**

### CODE REVIEW

File: lib/core/router/go_router.dart [hunk 1]
```diff
@@ -41,6 +43,16 @@ class Routing {
             name: AppRoute.result,
             builder: (context, state) => const ResultPage(),
           ),
+          GoRoute(
+            path: '/${AppRoute.progressChart}',
+            name: AppRoute.progressChart,
+            builder: (context, state) => const ProgressChartPage(),
```
Comment (saurabh-ifmt) [inline lib/core/router/go_router.dart]: remove progress chart page

File: lib/core/utils/color_utils.dart [hunk 2]
```diff
@@ -3,4 +3,12 @@ import 'package:flutter/material.dart';
 class ColorUtils {
   static ColorScheme get(BuildContext context) => Theme.of(context).colorScheme;
   static Color onBoarding = Color(0xFF22CCB2);
+  static Color resultsLow = Color(0xFFABE37E);
+  static Color resultsMedium = Color(0xFFFF7E42);
+  static Color resultsHigh = Color(0xFFE8618C);
+  static Color black = Color(0xFF000000);
+  static Color graphBgColor = Color(0xFF878CED);
+  static Color white = Color(0xFFFFFFFF);
```
Comment (saurabh-ifmt) [inline lib/core/utils/color_utils.dart]: remove all the colors

File: lib/core/widget/cases_dropdown_widget.dart [hunk 3]
```diff
@@ -0,0 +1,94 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import '../../../../../core/utils/color_utils.dart';
```
Comment (saurabh-ifmt) [inline lib/core/widget/cases_dropdown_widget.dart]: import file name

File: lib/core/widget/cases_search_widget.dart [hunk 4]
```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import '../../../../../core/utils/color_utils.dart';
```
Comment (saurabh-ifmt) [inline lib/core/widget/cases_search_widget.dart]: change import

File: pubspec.yaml [hunk 5]
```diff
@@ -37,8 +38,8 @@ flutter:
 
   # To add assets to your application, add an assets section, like this:
   assets:
-    - assets/ai_model/dental_classification_model.tflite
-    - assets/ai_model/model.tflite
+#    - assets/ai_model/dental_classification_model.tflite
+#    - assets/ai_model/model.tflite
```
Comment (saurabh-ifmt) [inline pubspec.yaml:42]: remove uncomment

File: lib/features/app/presentation/pages/view_analysis_page.dart [hunk 6]
```diff
@@ -0,0 +1,456 @@
+import 'package:flutter/material.dart';
+import 'package:dental_ui/core/utils/responsive_utils/responsive_padding.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:go_router/go_router.dart';
+
+import '../../../../../core/router/app_route.dart';
+import '../../../../../core/utils/color_utils.dart';
```
Comment (saurabh-ifmt) [inline lib/features/app/presentation/pages/view_analysis_page.dart]: remove

File: lib/features/app/presentation/pages/progress_chart_page.dart [hunk 7]
```diff
@@ -0,0 +1,210 @@
...
+  }
+
+  Widget _buildReportTitle(BuildContext context, String title) {
+    return Text(
+      title,
+      style: Theme.of(context).textTheme.titleLarge,
+    );
+  }
+
+  // Widget _buildToggleButtons(bool isWeekly, Function(bool) onToggle) {
```
Comment (saurabh-ifmt) [inline lib/features/app/presentation/pages/progress_chart_page.dart]: note required then remove

File: lib/features/app/presentation/pages/bottom_tab/result_page.dart [hunk 8]
```diff
@@ -1,10 +1,479 @@
...
+              weeklyData: weeklyData,
+              monthlyData: monthlyData,
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+
+  // Widget _buildToggleButtons() {
```
Comment (saurabh-ifmt) [inline lib/features/app/presentation/pages/bottom_tab/result_page.dart]: remove comment

File: lib/features/app/presentation/pages/bottom_tab/result_page.dart [hunk 9]
```diff
@@ -1,10 +1,479 @@
+import 'package:dental_ui/core/utils/color_utils.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:fl_chart/fl_chart.dart';
+import 'package:go_router/go_router.dart';
+import '../../../../../core/router/app_route.dart';
```
Comment (saurabh-ifmt) [inline lib/features/app/presentation/pages/bottom_tab/result_page.dart]: import fix

File: lib/features/app/presentation/pages/bottom_tab/ai_analysis_page.dart [hunk 10]
```diff
@@ -1,10 +1,453 @@
 import 'package:flutter/material.dart';
+import 'package:dental_ui/core/utils/responsive_utils/responsive_padding.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:go_router/go_router.dart';
+
+import '../../../../../core/router/app_route.dart';
```
Comment (saurabh-ifmt) [inline lib/features/app/presentation/pages/bottom_tab/ai_analysis_page.dart]: code inport fix

File: lib/core/widget/cases_search_widget.dart [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+    this.onSearchPressed,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      width: 115,
+      child: TextField(
+        decoration: InputDecoration(
+          labelText: 'Cases',
```
Comment (saurabh-ifmt) [inline lib/core/widget/cases_search_widget.dart]: hide text property

File: lib/core/widget/cases_search_widget.dart [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+          fillColor: Colors.grey[300],
+          border: OutlineInputBorder(
+            borderRadius: BorderRadius.circular(32),
+            borderSide: BorderSide.none,
+          ),
+          contentPadding: const EdgeInsets.symmetric(vertical: 8, horizontal: 8),
+          suffixIcon: IconButton(
+            icon: SvgPicture.asset(
+              IconConstant.searchIcon,
+              colorFilter: ColorFilter.mode(Colors.black, BlendMode.srcIn),
```
Comment (saurabh-ifmt) [inline lib/core/widget/cases_search_widget.dart]: Colors use theme

File: lib/core/widget/chart_widget.dart [hunk 13]
```diff
@@ -0,0 +1,147 @@
...
+      child: Column(
+        children: [
+          Row(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              ChoiceChip(
+                label: Text(
+                  "Weekly",
+                  style: TextStyle(
+                    color: isWeekly ? ColorUtils.get(context).primary.withOpacity(0.6) : Colors.white,
```
Comment (saurabh-ifmt) [inline lib/core/widget/chart_widget.dart]: user surface color

File: lib/core/widget/chart_widget.dart [hunk 14]
```diff
@@ -0,0 +1,147 @@
...
+                  borderRadius: BorderRadius.circular(30),
+                  side: BorderSide.none,
+                ),
+                showCheckmark: false,
+              ),
+              ChoiceChip(
+                label: Text(
+                  "Monthly",
+                  style: TextStyle(
+                    color: !isWeekly ? ColorUtils.get(context).primary.withOpacity(0.6) : Colors.white,
```
Comment (saurabh-ifmt) [inline lib/core/widget/chart_widget.dart]: fix color

File: lib/core/widget/cases_search_widget.dart (hunk 11 above)
Comment (AdityaSanap1821) [reply lib/core/widget/cases_search_widget.dart]: used hint text, so once user starts typing the hint disappear.

File: lib/core/widget/chart_widget.dart (hunk 14 above)
Comment (AdityaSanap1821) [reply lib/core/widget/chart_widget.dart]: used color scheme as specified

**Pull request merged**